tests/test_C0_data_processing.py

```diff
@@ -1131,6 +1132,26 @@ def test_compute_timeseries_properties_TIMESERIES_not_in_dict_asset():
     ), f"The function has changed the dict_asset to {dict_asset}, eventhough it should not have been modified and stayed identical to {dict_exp}."
 
 
+def test_compute_timeseries_properties_TIMESERIES_not_in_dict_asset():
+    dict_asset = {
+        UNIT: "str",
+        LABEL: "str",
+    }
+    dict_exp = copy.deepcopy(dict_asset)
+    assert (
```

Comment:
this test belongs to the package `copy.deepcopy` I would say

Reply:
Weird, I did not even add that here... where does that come from?